libusb/os/windows_common.h

```diff
@@ -318,8 +336,7 @@ struct winusb_transfer_priv {
 struct windows_backend {
 	int (*init)(struct libusb_context *ctx);
 	void (*exit)(struct libusb_context *ctx);
-	int (*get_device_list)(struct libusb_context *ctx,
-		struct discovered_devs **discdevs);
+	int (*get_device_list)(struct libusb_context *ctx);
```

Comment:
We should add a comment above this function 

```
// List all devices connected to the system and update the `hotplug_status` field of all known devices.
int (*get_device_list)(struct libusb_context *ctx);
```

Reply:
ok